test/e2e/instrumentation/monitoring/cadvisor.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2017 The Kubernetes Authors.
```

Comment:
It could be under either.  As it currently only checks to see if the endpoint is reachable, I think placing it under instrumentation would be more appropriate.